### DISCUSSION TITLE
When server not enable

### DISCUSSION THREAD

**Discussion by lloricode** [spatie/ray#564] [Ideas]
Hi!

Is there a way to stop receiving data when server is disabled?

![Screenshot from 2021-10-04 10-22-00](https://user-images.githubusercontent.com/8251344/135784332-972ac684-1384-49c6-b4e3-8851e338d8ec.png)

even not enable server, still getting data from server

Thank you

**Comment by freekmurze** [comment]
I'm thinking we already do this, when the connection is stopped in UI, we stop the underlying SSH connection, and no new items should come in Ray. Right @AdrianMrn ?

**Reply by lloricode** [reply]
Hmm it seems not working on my side, since I always forgot to connect/enable the server connection in ray app, but still receiving the data in app
